index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/0eaab31a39.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/css/style.css">
+    <title>Apple homepage</title>
+</head>
+
+<body>
+   <div class="grid-container">
+       <div class="main flex-col">
+           <div class="nav-bar flex-row">
+                <span class="nav-item"><i class="fab fa-apple"></i></span>
+                <span class="nav-item"><a href="#">Store</a></span>
+                <span><a href="#">Mac</a></span>
+                <span><a href="#">iPod</a></span>
+                <span><a href="#">iPhone</a></span>
+                <span><a href="#">iPad</a></span>
+                <span><a href="#">iTunes</a></span>
+                <span><a href="#">Support</a></span>
+                <span class="nav-item mag-span flex-row">
+                    <span class="mag">
+                        <img src="./images/mag.png" alt="">
+                    </span>
+                    <input type="text" name="search" class="nav-search-input">
+                </span>
+           </div>
+           <div class="titles flex-col">
+                <img class="h1" src="images/banner-h1.PNG" alt="">
+                <img class="h2" src="images/banner-h2.PNG" alt="">
+                <img class="h3" src="images/banner-h3.PNG" alt="">
+            </div>
+       </div>
+       <div class="promo">
+            <div class="image-wrap flex-row">
+                <img class="promo-img" src="./images/image1.jpg" alt="">
+                <img class="promo-img" src="./images/image2.jpg" alt="">
+                <img class="promo-img" src="./images/image3.jpg" alt="">
+                <img class="promo-img" src="./images/image4.jpg" alt="">
+            </div>
```

Comment:
- [ ] These 4 images should be links ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))